### ISSUE TITLE
https is redirected to http

### ISSUE THREAD

**Issue by romanchyla** [adsabs/ADSCore#23]
see the following request/response:

```curl -I https://ui.adsabs.harvard.edu/abs/arXiv:1511.06066
HTTP/1.1 301 MOVED PERMANENTLY
Date: Wed, 04 Dec 2019 23:58:34 GMT
Content-Type: text/html; charset=utf-8
Content-Length: 273
Connection: keep-alive
Server: openresty/1.15.8.1
Location: http://ui.adsabs.harvard.edu/abs/2015arXiv151106066W/abstract
X-RateLimit-Limit: 400
X-RateLimit-Remaining: 363
X-RateLimit-Reset: 1575587136
Retry-After: 83221
Vary: Cookie
Set-Cookie: session-core=.eJwVzVtygjAAQNG95FsdglEefyggj_qgppX4wwAJGkCRJjTajntvXcC95xfkgzwD-xfkZcmEyGTXsCuwwUKEfMqILNv5bmmZH_3KVAdjVj1cRKeBkR4iHhvKC8EIFJ0EdpW3go0Au9_4F8v4a6Fr0BpDfazNsD61EbI1bQIRnCEDPEeg7LqGM_GixT_Mu1cyYZGitfTiU-2Q99Zss4v8xp5G6NIng3T8TuxNj-5ucA37vMN8eFBWpe4peEO-aPr7Kdn2Ze0pt4gvizfsoKQJeUg3iKfBZkl4sb3oY6-6Jo1Fi413nGdVgAl5oP5iXlcBnboUrwfruFNhDgv8mX43mtmr5Kc0nLr7CN2gl1srGw9fdy379I_7d90P5iW-yfXqJ-KyecQnhRMlU3MRUJK1mdK9ZJKycxgnk73p4LKO7gGUaeoQF63gXkQ3BqM1eD7_AAy6gYs.XehIKg.h5PKBSIirwOn0-Z-bRZIY8Bb6G0; HttpOnly; Path=/
```

the problem is likely on this line: https://github.com/adsabs/ADSCore/blob/c02ced2f3c0cb0795ac039233aa477fdb7bd6b4c/adscore/routes.py#L266

if it returns relative url, the server is already running on http; not sure if nginx needs to be told to turn that properly into https; also the links are relative

**Comment by romanchyla** [comment]
@marblestation please see this issue; currently it makes one of adsrex tests fail

**Comment by marblestation** [comment]
This is closed by 768d28ea9f85dd8e612c9cbfed019dfe329a77a2
